### ISSUE TITLE
Mobile auto-complete

### ISSUE THREAD

**Issue by HStewart23** [fac-15/I-Choose-Who-#27]
So... this is an obvious stretch thing because you have hit all the project goals. Basically when I try to look at your app on my mobile the auto complete does not show up. If I type in a pokemon name it will bring up the image but I don't get the dropdown or any other form of auto complete functionality.

![moblie gif](https://media.giphy.com/media/vIg5t0KHYqgE0/giphy.gif)

**Comment by aterna01** [comment]
Yeah, you are right. It looks like Safari doesn't like Datalist <option> tag. We found a workaround online.
Just applied it.
